### ISSUE TITLE
argument processing

### ISSUE THREAD

**Issue by jelovirt** [dita-ot/dita-ot#347]
Converted from [SourceForge issue 2052043](http://sourceforge.net/support/tracker.php?aid=2052043), submitted by tgrantham

Using java command:

java -jar c:/dita-ot1.4.2.1/lib/dost.jar /i:RogosinUserManual.ditamap /outdir:pdf /transtype:pdf /filter:ditaval.pdf.xml /ditadir:c:/dita-ot1.4.2.1

results in:

BUILD FAILED
C:\dita-ot1.4.2.1\build.xml:55: The following error occurred while executing this line:
C:\dita-ot1.4.2.1\build_preprocess.xml:195: C:\Documents and Settings\tgrantham\My Documents\Rogosin User Guide\${user.input.dir} not found.

Whereas java command:

java -jar c:/dita-ot1.4/lib/dost.jar /i:RogosinUserManual.ditamap /outdir:pdf /transtype:pdf /filter:ditaval.pdf.xml /ditadir:c:/dita-ot1.4

Succeeds.

In other words, dita-ot1.4 release works, dita-ot1.4.2.1 does not.
